addons/TBMod_main/configs/Cfg3DEN.hpp

```diff
@@ -126,14 +126,14 @@ class Cfg3DEN
                     class TBMod_main_keys
                     {
                         displayName = "Schlüssel für das Auto";
-                        tooltip = "Gibt der KI einen Schlüssel für das Auto, muss der Variablenname sein!";
+                        tooltip = "Gibt der KI mehere Schlüssel für Fahrzeuge, muss der Variablenname sein ([\"car1\",\"car2\"])!";
```

Comment:
Dies wird nicht wie erwartet ["car1","car2"]] angezeigt, sondern [\car1\,\car2\]. Was ist der Fehler? @shukari

Reply:
Du machst in einem String neue auf entweder "" oder ' in Strings verwenden

Reply:
So escapen geht in arma3 nicht